lib/Job.js

```diff
@@ -4,6 +4,7 @@ const events = require('events')
 const cronParser = require('cron-parser')
 const CronDate = require('cron-parser/lib/date')
 const sorted = require('sorted-array-functions')
+const rruleUtil = require('./utils/rruleUtil');
```

Comment:
this should be dynamically required on the first attempt to add an rrule.

Reply:
As it is a utility, I disagree. Since we handle the missing dependency then we can include the util to sit and wait. Your call, let me know if I should change it

Reply:
I would rather see it not as merely a set of utility functions, but more like a module, almost a plugin, that should encapsulate as much of RRule functionality as possible, and should be as absent as possible if user does not opt in to use it.